api/src/test/unit/java/uk/gov/dvsa/mot/vehicle/read/core/VehicleReadServiceDatabaseTest.java

```diff
@@ -471,6 +471,89 @@ public void findByDvlaVehicleId_ReturnsEmptyListWhenNoVehiclesWereFound() {
 
+    @Test
+    public void mapDvlaVehicleSqltoJson_ReturnsDvlaMakeAndModel() {
+
+        uk.gov.dvsa.mot.persist.model.DvlaVehicle persistedDvlaVehicle = createPersistedDvlaVehicle();
+
+        DvlaVehicle dvlaVehicle = vehicleReadServiceDatabase.mapDvlaVehicleSqltoJson(persistedDvlaVehicle);
+
+        assertEquals(dvlaVehicle.getRegistration(), DEFAULT_REGISTRATION);
+        assertEquals(dvlaVehicle.getColour1(), DEFAULT_COLOR_NAME);
+        assertEquals(dvlaVehicle.getColour2(), DEFAULT_COLOR_NAME);
+        assertEquals(dvlaVehicle.getFuelType(), DEFAULT_FUEL_TYPE_NAME);
+        assertEquals(dvlaVehicle.getBodyTypeCode(), DEFAULT_BODY_TYPE_CODE);
+        assertEquals(dvlaVehicle.getMakeDetail(), DEFAULT_MAKE_NAME);
+        assertEquals(dvlaVehicle.getModelDetail(), DEFAULT_MODEL_DETAIL_NAME);
+        assertNull(dvlaVehicle.getMakeInFull());
+    }
+
+    @Test
+    public void mapDvlaVehicleSqltoJson_ReturnsDvsaMakeAndModel() {
+
+        uk.gov.dvsa.mot.persist.model.DvlaVehicle persistedDvlaVehicle = createPersistedDvlaVehicle();
+
+        persistedDvlaVehicle.setDvsaMake("DVSA_MAKE");
+        persistedDvlaVehicle.setDvsaModel("DVSA_MODEL");
+
+        DvlaVehicle dvlaVehicle = vehicleReadServiceDatabase.mapDvlaVehicleSqltoJson(persistedDvlaVehicle);
+
+        assertEquals(dvlaVehicle.getRegistration(), DEFAULT_REGISTRATION);
+        assertEquals(dvlaVehicle.getColour1(), DEFAULT_COLOR_NAME);
+        assertEquals(dvlaVehicle.getColour2(), DEFAULT_COLOR_NAME);
+        assertEquals(dvlaVehicle.getFuelType(), DEFAULT_FUEL_TYPE_NAME);
+        assertEquals(dvlaVehicle.getBodyTypeCode(), DEFAULT_BODY_TYPE_CODE);
+        assertEquals(dvlaVehicle.getMakeDetail(), "DVSA_MAKE");
+        assertEquals(dvlaVehicle.getModelDetail(), "DVSA_MODEL");
+        assertNull(dvlaVehicle.getMakeInFull());
+    }
+
+    @Test
+    public void mapDvlaVehicleSqltoJson_ReturnsDvlaMakeInFull() {
+
+        uk.gov.dvsa.mot.persist.model.DvlaVehicle persistedDvlaVehicle = createPersistedDvlaVehicle();
+
+        persistedDvlaVehicle.setDvsaMake("");
+        persistedDvlaVehicle.setDvsaModel("");
+        persistedDvlaVehicle.getMakeDetail().setName("unknown");
+        persistedDvlaVehicle.getModelDetail().setName("UNKNOWN");
+        persistedDvlaVehicle.setMakeInFull("MAKE_IN_FULL");
+
+        DvlaVehicle dvlaVehicle = vehicleReadServiceDatabase.mapDvlaVehicleSqltoJson(persistedDvlaVehicle);
+
+        assertEquals(dvlaVehicle.getRegistration(), DEFAULT_REGISTRATION);
+        assertEquals(dvlaVehicle.getColour1(), DEFAULT_COLOR_NAME);
+        assertEquals(dvlaVehicle.getColour2(), DEFAULT_COLOR_NAME);
+        assertEquals(dvlaVehicle.getFuelType(), DEFAULT_FUEL_TYPE_NAME);
+        assertEquals(dvlaVehicle.getBodyTypeCode(), DEFAULT_BODY_TYPE_CODE);
+        assertEquals(dvlaVehicle.getMakeDetail(), "unknown");
+        assertEquals(dvlaVehicle.getModelDetail(), "UNKNOWN");
+        assertEquals(dvlaVehicle.getMakeInFull(), "MAKE_IN_FULL");
```

Comment:
No big deal, but I would prefer 'UNKNOWN' as a const as it's used in several places.